sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Release History
 
 ## 1.4.0b3 (Unreleased)
+- First preview of new API for authenticating users with `DeviceCodeCredential`
+  and `InteractiveBrowserCredential`
+  - new method `authenticate` authenticates a user, returns a serializable
```

Comment:
Should this be "interactively authenticates a user"... so it matches with the description of `disable_automatic_authentication` below?